docs/docker/docker-image-size.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+set +x
+
+# This script computes **compressed image size with all its layers**.
+# This solution is based on https://stackoverflow.com/a/55156181/314883
+
+DOCKER_IMAGE_NAME="docker-image-to-count-compressed-size"
+FILE_NAME="docker-image-to-count-compressed-size"
+RUN_AS=""
+if [[ "$(uname)" == "Darwin" ]]; then
+  RUN_AS=""
```

Comment:
Let's drop this? User can always add themselves to the docker group and be able to manipulate images without sudo.

Reply:
Done